clusters/app.ci/prow-monitoring/Makefile

```diff
@@ -1,147 +1,20 @@
-	oc apply -f ./grafana_rbac.yaml --as=system:admin
-
-.PHONY : prow-monitoring-cleanup
-prow-monitoring-cleanup:
-	@echo "Cleaning up prow-monitoring stack ..."
-	oc process -f ./prometheus_expose.yaml | oc delete --ignore-not-found=true --as=system:admin -f -
-	oc delete -f ./mixins/prometheus_out/prometheus_rule_prow.yaml --ignore-not-found=true --as=system:admin
-	oc delete -f ./prometheus_crd.yaml --ignore-not-found=true --as=system:admin
-	oc delete -f ./sinker_service_monitor.yaml --ignore-not-found=true --as=system:admin
-	oc delete -f ./ghproxy_service_monitor.yaml --ignore-not-found=true --as=system:admin
-	oc delete -f ./hook_service_monitor.yaml --ignore-not-found=true --as=system:admin
-	oc delete -f ./deck_service_monitor.yaml --ignore-not-found=true --as=system:admin
-	oc delete -f ./deck-internal_service_monitor.yaml --ignore-not-found=true --as=system:admin
-	oc delete -f ./jenkins-operator_service_monitor.yaml --ignore-not-found=true --as=system:admin
-	oc delete -f ./plank_service_monitor.yaml --ignore-not-found=true --as=system:admin
-	oc delete -f ./exporter_service_monitor.yaml --ignore-not-found=true --as=system:admin
-	oc delete -f ./tide_service_monitor.yaml --ignore-not-found=true --as=system:admin
-	oc delete -f ./prometheus_service_monitor.yaml --ignore-not-found=true --as=system:admin
-	oc delete -f ./alert_manager_service_monitor.yaml --ignore-not-found=true --as=system:admin
-	oc delete -f ./grafana_service_monitor.yaml --ignore-not-found=true --as=system:admin
-	oc delete -f ./prometheus_operator_deploy.yaml --ignore-not-found=true --as=system:admin
-	oc delete -f ./prometheus_rbac.yaml --ignore-not-found=true --as=system:admin
-	oc delete -f ./alert_manager_crd.yaml --ignore-not-found=true --as=system:admin
-	oc delete secret -n prow-monitoring alertmanager-prow --ignore-not-found=true --as=system:admin
-	oc process -f ./alert_manager_expose.yaml | oc delete --ignore-not-found=true --as=system:admin -f -
-	oc delete -f ./alert_manager_rbac.yaml --as=system:admin
-	oc delete -f ./grafana_deploy.yaml --ignore-not-found=true --as=system:admin
-	oc delete configmap -n prow-monitoring grafana-dashboard-definitions-0 --ignore-not-found=true --as=system:admin
-	oc delete configmap -n prow-monitoring grafana-dashboards --ignore-not-found=true --as=system:admin
-	oc delete configmap -n prow-monitoring grafana-datasources --ignore-not-found=true --as=system:admin
-	oc delete secret -n prow-monitoring grafana-config --ignore-not-found=true --as=system:admin
-	oc process -f ./grafana_expose.yaml | oc delete --ignore-not-found=true --as=system:admin -f -
-	oc delete -f ./grafana_rbac.yaml --ignore-not-found=true --as=system:admin
-	oc delete -f ./client_rbac.yaml --ignore-not-found=true --as=system:admin
-	oc delete -f ./ci_hook_rbac.yaml --ignore-not-found=true --as=system:admin
-	oc delete -f ./prometheus_project.yaml --wait=true --ignore-not-found=true --as=system:admin
-
-
-.PHONY : grafana-debug-deploy
```

Comment:
Why remove this one?

Reply:
It is deployed by `applyconfig`.
I have removed "_" prefix in the folder name of `debug`.